contracts/src/token/erc20/extensions/burnable.rs

```diff
@@ -168,7 +168,7 @@ mod tests {
             .set(one);
 
         let result = contract.burn_from(Address::ZERO, one);
-        assert!(matches!(result, Err(Error::InvalidSender(_))));
+        assert!(matches!(result, Err(Error::InvalidApprover(_))));
```

Comment:
Right now it looks weird with the test name. Maybe better test name?

Reply:
Changed `*_sender` to `*_approver` that is not ideal though, especially when names here follows our previous error convention "burns_from_errors_when_invalid_approver"
I think we should later refactor test names for `erc20`